src/components/Image.test.js

```diff
@@ -21,10 +22,15 @@ describe('<Image />', () => {
       expect(wrapper.find('img').prop('alt')).toEqual(defaultProps.image.name)
     })
 
-    it('should call onDoubleClick from props when image is doubleClicked', () => {
+    it('render with correct id prop', () => {
       const wrapper = shallow(<Image {...defaultProps}/>)
-      wrapper.find('img').simulate('doubleClick')
-      expect(defaultProps.onDoubleClick).toHaveBeenCalled()
+      expect(wrapper.find('img').prop('id')).toEqual(defaultProps.image.id)
     })
+
+    // it('should call onDoubleClick from props when image is doubleClicked', () => {
```

Comment:
I think you want to test this right? (something like it at least)

Reply:
Yes I have uncommented it :)